examples/opentelemetry/Dockerfile-opentelemetry

```diff
@@ -1,11 +1,9 @@
 FROM alpine:3.19@sha256:c5b1261d6d3e43071626931fc004f70149baeba2c8ec672bd4f27761f8e1ad6b as otelc_curl
-RUN apk --update add curl
+    RUN apk --update add curl
 
-FROM otel/opentelemetry-collector:latest@sha256:bba8474ea69d11f0450e38c50040efea47f4190723a9fb64cc9d44939b4ade4c
-
-COPY --from=otelc_curl / /
-
-COPY ./otel-collector-config.yaml /etc/otel-collector-config.yaml
-USER 0
-RUN chmod o+r /etc/otel-collector-config.yaml
-USER nobody
+FROM otel/opentelemetry-collector:0.99.0@sha256:bba8474ea69d11f0450e38c50040efea47f4190723a9fb64cc9d44939b4ade4c
```

Comment:
this is the same image

im not adverse to using the version explicitly tho

Reply:
Same image, different version. The collector is not stable yet. For example, the rename of `logging` to `debug` exporter that is commented on elsewhere in this PR.

Reply:
my point is the sha hasnt changed - ie bba8474ea69d11f0450e38c50040efea47f4190723a9fb64cc9d44939b4ade4c

when you set the sha docker pretty much ignores everything else anyway

Reply:
Yes, I see. It's a bit confusing IMO to use a `latest` with SHA-pinning, but frankly I can go either way.

Reply:
if you want to separate out that change i would be happy to review - i think its more explicit, and depending on how upstream releases can reduce update noise